dev-client/src/screens/SoilScreen/utils/renderValues.ts

```diff
@@ -54,9 +54,9 @@ export const pitMethodSummary = (
     summary = t(`soil.texture.rockFragment.${soilData.rockFragmentVolume}`);
   } else if (
     method === 'soilColor' &&
-    soilData.colorHue &&
-    soilData.colorValue &&
-    soilData.colorChroma
+    typeof soilData.colorHue === 'number' &&
```

Comment:
comment: see my request for `ColorScreen.tsxt` -- maybe a common "isNumericColor" or similar function could be extracted so we don't have to repeat this pattern of data checks in more places.